pkg/loop/internal/core/services/capability/capabilities_registry.go

```diff
@@ -296,14 +304,25 @@ func (c *capabilitiesRegistryServer) ConfigForCapability(ctx context.Context, re
 
 	ccp := &capabilitiespb.CapabilityConfig{
 		DefaultConfig: ecm,
-		RemoteConfig: &capabilitiespb.CapabilityConfig_RemoteTriggerConfig{
+	}
+
+	if cc.RemoteTriggerConfig != nil {
```

Comment:
This feels like we could move it into a switch statement: I'd expect one of these to apply for a given capability, not both